_data/media.yml

```diff
@@ -0,0 +1,30 @@
+---
+- title: "[Video] Vitalik dancing badger dance"
```

Comment:
Can we replace these with live ones?

    Media


Some links:

https://www.youtube.com/watch?v=6lLT33tsJjs
https://www.youtube.com/watch?v=lUDy1MoeYnI
https://www.youtube.com/watch?v=s0ATpQ4_XFc
https://www.youtube.com/watch?v=zxN-PqYkZ1M
https://www.youtube.com/watch?v=S782Ppzvkd0
https://www.youtube.com/watch?v=rQOp3qoDF0g

Reply:
Links and data were updated.
Would you like to add media section back?
According to the opened [issue#23](https://github.com/vacp2p/vac.dev/issues) media section and content [were hided](https://github.com/vacp2p/vac.dev/pull/28).
Also new design assumes a preview picture for a video. What pictures for the videos would you like to use?

Reply:
Oh right, miscommunication between me and Jonny I think. Yes please, we can use whatever preview is default in youtube or so I think

Reply:
Should the video open as a modal on the page, or should it just be a YouTube link and opened in a new tab?